src/ZendCodingStandard/CodingStandard.php

```diff
@@ -0,0 +1,130 @@
+<?php
+namespace ZendCodingStandard;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Util\Common;
+use PHP_CodeSniffer\Util\Tokens;
+
+use function preg_match;
+use function sprintf;
+use function strpos;
+use function strtolower;
+
+class CodingStandard
+{
+    public static $allowedTypes = [
```

Comment:
It was here, because the whole it was copied from Common class from PHP_CodeSniffer. It is removed now as it was used only in one place. I've moved it to the property in sniff which was using it. See ffac5b6